infra/lb.tf

```diff
@@ -0,0 +1,115 @@
+resource "aws_security_group" "lb" {
+  name        = "${var.name_prefix}-load-balancer"
+  description = "Production load balancer"
+  vpc_id      = aws_vpc.this.id
+
+  ingress {
```

Comment:
are you forwarding port 80 to 443? Otherwise port 80 will just run into nothingness here

Reply:
Ah that was for testing. If we don't have LB listener on 80 does the SG rule matter?

Reply:
it will, because when we look at the monitoring we will be very confused by all the 502 bad gateway errors that happen when someone tries to connect to port 80 on the site. I would just [redirect](https://aws.amazon.com/premiumsupport/knowledge-center/elb-redirect-http-to-https-using-alb/)

Reply:
Sounds good, thanks!